vars/artifactSetVersion.groovy

```diff
@@ -93,7 +93,7 @@ enum GitPushMode {NONE, HTTPS, SSH}
     /** Defines the template for the automatic version which will be created. */
     'versioningTemplate',
     /** Controls which protocol is used for performing push operation to remote repo.
-      * Required credentials needs to be configured ('gitSshKeyCredentialsId'/'TBD').
+      * Required credentials needs to be configured ('gitSshKeyCredentialsId'/'config.gitHttpsCredentialsId').
```

Comment:
Why has this parameter a leading `config.` and the other one not?

Reply:
mistake c&p. thanks, I will fix it!